compiler/luci-interpreter/src/kernels/Softmax.cpp

```diff
@@ -60,5 +75,17 @@ void Softmax::evalFloat() const
                                  getTensorShape(output()), getTensorData<float>(output()));
 }
 
+template <typename T> void Softmax::evalQuantized() const
+{
+  tflite::SoftmaxParams params{};
+  // params.beta = _params.beta;
```

Comment:
What is this comment?

Reply:
Oh, this part is not needed on hear. I will remove it.